plans/step9-plan.md

```
@@ -0,0 +1,283 @@
+- `ADMIN_SESSION_SECRET`/`ADMIN_CSRF_SECRET` 교체 시 기존 세션 무효화 정책을 운영 문서에 명시한다.
+- 로그인/2차 인증 실패 누적은 구조화 로그로 모니터링하고, 임계치 초과 시 알림 기준을 별도 정의한다.
+
+#### 예정 테스트
+
+1. **관리자 로그인 1차 성공 → 2차 인증 챌린지 발급**
+   ```bash
+   curl -i -c /tmp/admin-cookie.txt -X POST http://localhost:3000/api/admin/auth/login \
+     -H "Content-Type: application/json" \
+     -d '{"username":"admin","password":"<ADMIN_PASSWORD>"}'
+   ```
+   - 기대 결과: HTTP `200`, `requiresTwoFactor=true` 응답
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)